### PR TITLE
Engineer Component Front

### LINKED ISSUE #8: [frontend] engineer component
image

### PR DESCRIPTION
**Pull request by braaAwni**
image

### CODE REVIEW

**Review by mo7amedehab97 — DISMISSED** [review]
nice work braa'

**Review by nabilramy — DISMISSED** [review]

File: client/src/Component/EngineerCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <p>Architect & Engineer</p> 
+  </div>
+</div>
+<div class="card">
+  <img src="http://themesflat.com/html/nah/images/member/1t1.png" alt="Avatar" />
+  <div className='Engdetailes'>
+    <h4><b>John Doe</b></h4> 
+    <p>Architect & Engineer</p> 
+  </div>
+</div>
```
Comment (aaamra) [inline client/src/Component/EngineerCard/index.jsx]: extract the `card` to a component

**Review by aaamra — CHANGES_REQUESTED** [review]

**Review by nabilramy — DISMISSED** [review]

**Review by mo7amedehab97 — DISMISSED** [review]

**Review by aaamra — DISMISSED** [review]

**Review by KAHMOOSHA — DISMISSED** [review]

File: client/src/App.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
-import Home from './Component/Home';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from './Pages/Home';
 import NotFoundPage from './Pages/NotFound';
 
+
 function App() {
   return (
     <div className="App">
```
Comment (mo7amedehab97) [inline client/src/App.js]: where is the app

**Review by mo7amedehab97 — CHANGES_REQUESTED** [review]
you deleted the pervious work 
updated the main and merge it with your branch then fit your current work with our previous work

**Review by mo7amedehab97 — APPROVED** [review]

**Review by aaamra — CHANGES_REQUESTED** [review]
fix your eslint errors

**Review by nabilramy — APPROVED** [review]

**Review by aaamra — APPROVED** [review]

**Review by mo7amedehab97 — APPROVED** [review]

**Pull request merged**